tests/Api/Controller/AbstractTestController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Flarum\Tests\Api\Controller;
+
+use Flarum\Http\Controller\ControllerInterface;
+use Flarum\Tests\Test\TestCase;
+use Flarum\User\User;
+use Psr\Http\Message\ResponseInterface;
+
+abstract class AbstractTestController extends TestCase
```

Comment:
Should this be `AbstractControllerTest`?

Reply:
.. so this way this class is automatically ignored by phpunit.

Reply:
But shouldn't phpunit ignore abstract classes anyway?

Reply:
or do https://stackoverflow.com/questions/5934347/mark-a-phpunit-test-class-as-to-be-ignored

Reply:
> But shouldn't phpunit ignore abstract classes anyway?

Yes. I think so as well. I'll refactor.

Reply:
Renamed to `ApiControllerTestCase` giving a proper indication what it is for.